src/PointerEvent.js

```diff
@@ -87,6 +87,35 @@ function PointerEvent(inType, inDict) {
   e.pointerType = inDict.pointerType || '';
   e.hwTimestamp = inDict.hwTimestamp || 0;
   e.isPrimary = inDict.isPrimary || false;
+
+  var rect;
+  function boundingClientRect() {
+    if (rect) {
+      return rect;
+    }
+    return (rect = inDict.target.getBoundingClientRect());
+  }
+
+  // CSSOM View Module's extension to mouse events
+  if (inDict.offsetX === undefined) {
+    Object.defineProperty(e, 'offsetX', {
+      get: function() {
+        return e.pageX - boundingClientRect().left;
```

Comment:
`offsetX` is relative to the padding edge of the target per [spec](http://www.w3.org/TR/cssom-view/#dom-mouseevent-offsetx):

> return the x-coordinate of the position where the event occurred relative to the origin of the padding edge of the target node, ignoring the transforms that apply to the element and its ancestors

However `getBoundingClientRect()` returns the rect of border edge, so I think the calculation is incorrect here.

Additionally, "ignoring the transforms" part seems also unimplemented (and I think this is quite tricky to polyfill).